code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -3,9 +3,9 @@
 	desc = "And boom goes the weasel."
 	icon_state = "explosive"
 	origin_tech = "materials=2;combat=3;biotech=4;syndicate=4"
-	var/weak = 1.6
-	var/medium = 0.8
-	var/heavy = 0.4
+	var/weak = 2
+	var/medium = 1
+	var/heavy = 0.5
```

Comment:
Uh, why are you making these bigger? They were at the previous values so that they wouldn't be maxcap unless you somehow got more than 10.